pages/tech-team.tsx

```diff
@@ -0,0 +1,124 @@
+import Layout from "../components/Layout";
+import OfficerJson from "../data/Officer.json";
+import OfficerIcon from "../components/OfficerIcon";
+import TechProject from "../components/TechProject";
+import ProjectsJson from "../data/TechTeamProjects.json";
+
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+type Props = {
+  techOfficerName: string;
+  techOfficerEmail: string;
+  imgPath: string;
+};
+
+const TeachTeamPage = ({
+  techOfficerEmail,
+  techOfficerName,
+  imgPath,
+}: Props) => (
+  <Layout title="Tech team page">
```

Comment:
```suggestion
  <Layout title="SHPE UTD">
```
Not sure if we should make the title SHPE UTD here as well - I think thats what we've been using on the other pages but honestly we can probably just hardcode the title in the Layout component and remove the title prop entirely if we want every pages title to be SHPE UTD but we could do something like `Tech Team - SHPE UTD` or `SHPE UTD | Tech Team` if we don't want the same title on every page